### ISSUE TITLE
Dialogs from NVDA take more time to initialize when invoked, if the Windows Magnifier is running

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#6924]
I've noticed it from some time, but only recently was able to track it down. If the Windows Magnifier is running, invoked dialogs of NVDA take some time to initialize. I'm also using the Windows "high-contrast black" theme, but I don't know if that matters. The magnifier is set to magnify to 700%, but I don't think the magnification level matters. In the magnifier's settings, all 3 tracking options are enabled. Color inversion is disabled. The view mode is "Full Screen". If needs be, I'll try to provide logs from NVDA while reproducing the issue.

Steps to reproduce:
1. Launch NVDA.
2. Launch the Windows Magnifier. This is most easily done by pressing the Windows logo key and the "Plus" key from the NumPad simultaneously.
3. Open the magnifier settings and enable the 3 options there related to tracking, then press OK in that dialog to close it and save the settings.
4. Increase the magnification level a few times, because now I've tested and it seams that the magnification level may affect if the bug occurs or not.
5. Invoke any dialog of NVDA (the method of invoking - via keyboard shortcut or from NVDA's menu seams not to matter).

Expected results:
The dialog should appear immediately and NVDA should start speaking its contents.

Actual results:
The dialog takes some time to initialize and NVDA starts speaking its contents after some delay.

Test environment:
- Operating system: Windows 10 Pro (build 14393.693), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-13899,af44c1d4, in bulgarian.
- Processor: Intel Core i5-2320 at 3.00GHz.
- RAM Memory: 8.00GB.
- Graphics: Palit GeForce GT 630, 1024MB dedicated memory + 3071MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC662 at Intel Cougar Point PCH - High Definition Audio Controller.

**Comment by ehollig** [comment]
@k-kolev1985, is this still an issue with NVDA 2017.3? It appears to work for me on Windows 10 when performing the above steps.

**Comment by k-kolev1985** [comment]
I've tried with a portable copy of NVDA 2017.3 and I don 't experience the issue. However, with the installed copy of NVDA next-14519,fd6dfb48, I can definitely reproduce the issue. I even tried with add-ons disabled and the issue still occurs.

**Comment by leonardder** [comment]
There is also a delay of like two seconds when opening the NVDA menu with zoom text running. Windows Magnifier is not ZOom Text, but this might still be related somehow.

**Comment by Adriani90** [comment]
@k-kolev1985, @leonardder, are there any improvements visible in 2019.1 regarding this performance issue? I am not using zoom text or magnification software at all, so I cannot really test this. Thanks in advance for your availability.

**Comment by k-kolev1985** [comment]
Hmm, I'm running an installed copy of NVDA version alpha-17024,97a70f1b and I don't experience this problem anymore. I tried it with a portable copy of NVDA 2019.1 and also did not experience the problem. Maybe whatever was causing the issue (either by NVDA or Windows - I don't know) was fixed.

**Comment by Adriani90** [comment]
Ok, then closing as works for me. If the issue occurs again, please comment on it and we can reopen it.
